File: src/main/java/kr/co/lunatalk/domain/image/dto/FindImageDto.java
package kr.co.lunatalk.domain.image.dto;


import kr.co.lunatalk.domain.image.domain.Image;

public record FindImageDto(
	String imageType,
	String imageUrl,
	String imageKey
) {
	public static FindImageDto from(Image image) {
		return new FindImageDto(image.getImageType().name(), image.getImagePath(), image.getImageKey());
	}
}

File: lunatalk-domain/src/main/kotlin/kr/co/lunatalk/domain/image/dto/FindImageDto.kt
package kr.co.lunatalk.domain.image.dto

import kr.co.lunatalk.domain.image.domain.Image

data class FindImageDto(
    val imageType: String,
    val imageUrl: String?,
    val imageKey: String?
) {
    companion object {
        fun from(image: Image): FindImageDto {
            return FindImageDto(
                imageType = image.imageType!!.name,
                imageUrl = image.imagePath,
                imageKey = image.imageKey
            )
        }
    }
}
